gsplat/cuda/_wrapper.py

```diff
@@ -422,6 +424,9 @@ def rasterize_to_pixels(
     if backgrounds is not None:
         assert backgrounds.shape == (C, colors.shape[-1]), backgrounds.shape
         backgrounds = backgrounds.contiguous()
+    if masks is not None:
+        assert masks.shape[0] == C, masks.shape
```

Comment:
maybe better to check it to be the same shape with `isect_offsets`

Reply:
fixed